docs/spec_draft.md

```diff
@@ -2527,6 +2528,68 @@ hidden state.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.rng_bit_generator
+
+### Semantics
+
+Returns an `output` filled with uniform random bits and an updated output state
+`output_state` given an initial state `initial_state` using the pseudorandom
+number generator algorithm `rng_algorithm`. The output is guaranteed to be
+deterministic function of `initial_state`, but it is not guaranteed to be
+deterministic between implementations.
+
+`rng_algorithm` is one of the following:
+  * `DEFAULT`: Backend specific algorithm.
```

Comment:
How about "Implementation-defined algorithm", to be in line with how we're calling similar things elsewhere in the spec.